### PR TITLE
Fix/docker compose docs

### PR DESCRIPTION
**Pull request by stephenmoloney**
I want to merge this change because...

There are many reports of issues with `docker-compose run`

***Please note: This is forked off of https://github.com/mirumee/saleor/pull/3117 so please wait for https://github.com/mirumee/saleor/pull/3117 first ***

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.

### CODE REVIEW

File: docs/customization/docker.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ Usage
 
    .. code-block:: bash
 
-    $ docker-compose up
+    $ docker-compose up -d
```
Comment (stephenmoloney) [inline docs/customization/docker.rst]: any opinions on this ?

Comment (ajostergaard) [reply docs/customization/docker.rst]: > any opinions on this `-d` ?

Don't you think it's likely to create issue reports saying "nothing happened :-S"?

Comment (stephenmoloney) [reply docs/customization/docker.rst]: that depends on whether you are using it for dev or production purposes.

Personally, I like to do `docker-compose up -d`

and then if I want to see logs,

```docker-compose logs -f`

Comment (ajostergaard) [reply docs/customization/docker.rst]: Agreed and me too but the doc is for development so not detaching might be more intuitive for beginners, don't you think?

Comment (patrys) [reply docs/customization/docker.rst]: @stephenmoloney I'd love to merge this but I also feel that for local development attached mode is more useful.

Comment (stephenmoloney) [reply docs/customization/docker.rst]: sure, I'll remove `-d`

**Comment by stephenmoloney** [comment]
Some more work to be done here as per discussion in https://github.com/mirumee/saleor/pull/3117

Will get to it next week hopefully. Closing the other PR.

**Comment by maarcingebala** [comment]
The change in Docker docs has been already made in another PR.

**Pull request closed without merging**